examples/dotenv-context/.env

```diff
@@ -0,0 +1,2 @@
+NAME=path
```

Comment:
something does not work well here. In the test we don't see the name of the context.
And can you add `foo` here too?
And can you et `production=0` in this file, and `production=1` in the `.env.local`?